### ISSUE TITLE
problem in Magento post-installation sequence

### ISSUE THREAD

**Issue by srgrbkl** [zepgram/magento2-fast-vm#110]
after deploy i get this  issues:

```
    m2: --- Magento post-installation sequence ---
    m2: Executing npm install...
    m2: OK
----------------------------------------------
    m2:   There are no commands defined in the "config" namespace.
    m2: 
    m2:   Did you mean one of these?
    m2:       app:config
    m2:       module:config
    m2:       setup:config
    m2:       setup:store-config
    ------------------------------------------
    m2:   There are no commands defined in the "deploy:mode" namespace.
    m2:   There are no commands defined in the "cache" namespace.
```

**Comment by zepgram** [comment]
When you have this error. Magento is not correctly installed. You probably had an error before. Can you give me your config.yml and the prompt during the Magento installation sequence ?

**Comment by srgrbkl** [comment]
config.yml:
```
vmconf:
  machine_name: 'm2'
  network_ip: '192.168.200.194'
  host_name: 'm2'
  memory: '4096'
  cpus: '3'
  mount: 'rsync'
  path: 'root'
  provision: 'all'
composer:
  username: '5d387ae439a7ce7ce2e47d0b1b334c48'
  password: '5d851b27af53dd2f4b686d244058abe5'
git:
  name: 'John Doe'
  email: 'john@doe.com'
  host: 'github.com'
  repository: 'ssh://git@github.com:project-name.git'
magento:
  url: 'dev.m2.com'
  source: 'composer'
  edition: 'community'
  version: '2.4.6'
  php_version: 'default'
  sample: 'true'
  mode: 'developer'
  currency: 'USD'
  language: 'en_US'
  time_zone: 'America/New_York'
  crypt_key: ''
  search_engine: 'elasticsearch'
  disable_two_factor_auth: 'true'
```
```
m2: --- Magento installation sequence ---
--------------------------------------------
m2: Installing search configuration...
    m2: 
    m2: In SearchConfig.php line 97:
    m2: 
    m2:   Search engine 'elasticsearch' is not an available search engine.
    m2: 
```

**Comment by zepgram** [comment]
Thanks, I find out the issue. Providing a fix in few minutes.

**Comment by zepgram** [comment]
Fix delivered on v1.7.1 : 
https://github.com/zepgram/magento2-fast-vm/commit/fea0b9a1b6add1a882087f0615fb5f1ebfcaf896

Thanks again for your feedback @srgrbkl

**Comment by srgrbkl** [comment]
> Fix delivered on v1.7.1 : [fea0b9a](https://github.com/zepgram/magento2-fast-vm/commit/fea0b9a1b6add1a882087f0615fb5f1ebfcaf896)
> 
> Thanks again for your feedback @srgrbkl

great, it works now!
